components/cost-management/staging/policies/kyverno_rbac.yaml

```
@@ -0,0 +1,21 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kyverno-admission-controller-update
+  labels:
+    rbac.kyverno.io/aggregate-to-admission-controller: "true"
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - namespaces
+  verbs:
+  - list
+  - get
+  - create
+  - delete
+  - update
+  - patch
+  - watch
```

definitely we don't need `create` and `delete` 

```suggestion
  - list
  - get
  - update
  - patch
  - watch
```

If I'm not mistaken, this is the minimum we should require
```suggestion
  - list
  - get
```